sdk/textanalytics/Azure.AI.TextAnalytics/src/Internal/ApiVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    [CodeGenModel("ApiVersion")]
+    internal readonly partial struct ApiVersion
```

Comment:
This is the only non-generated code we needed to add to make `ApiVersion` internal.